tailwind.config.ts

```diff
@@ -24,7 +24,8 @@ export default {
             colors: {
                 black: '#000000',
                 white: '#ffffff',
-                primary: '#312dfb'
+                primary: '#000000',
+                secondary: '#ffffff'
```

Comment:
Add trailing comma to improve future diffs of multi-line object declarations.

```suggestion
                primary: '#000000',
                secondary: '#ffffff',
```

Reply:
Boilerplate's prettier config has `trailingComma: "none"`